helpers/Login.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Dektrium project.
+ *
+ * (c) Dektrium project <http://github.com/dektrium/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace dektrium\user\helpers;
+
+use Yii;
+use yii\helpers\ArrayHelper;
+use dektrium\user\models\User;
+
+/**
+ * Login helper.
+ *
+ * @author Herbert Maschke <thyseus@gmail.com>
+ */
+class Login
```

Comment:
I'm not sure we need this a separate class. Maybe move it to the LoginForm and make there method `getLoginlist`.
Even if we need helper, I don't really like the name. `Login` is not displaying it's usage.